src/app/src/components/FilterSidebarFacilitiesTab.jsx

```diff
@@ -280,12 +284,16 @@ function FilterSidebarFacilitiesTab({
                                             to={{
                                                 pathname: makeFacilityDetailLink(
                                                     oarID,
+                                                    search,
                                                 ),
                                                 state: {
                                                     panMapToFacilityDetails: true,
                                                 },
                                             }}
-                                            href={makeFacilityDetailLink(oarID)}
+                                            href={makeFacilityDetailLink(
```

Comment:
This ensures we don't lose the query string when navigating to the facility details